maestro-client/src/test/java/maestro/PointTest.kt

```diff
@@ -22,21 +22,4 @@ internal class PointTest {
         // Then
         assertThat(point).isEqualTo(Point(1, 2))
     }
-
-    @Test
-    internal fun `Deserialize PointF`() {
```

Comment:
I think it should be fine to remove `PointF`, but heads up that there was some issue in Cloud that made me to add this test. Unfortunately I no longer remember what the issue was :)

Reply:
The way I see this is that IOSDriver.kt is responsible for mapping the maestro world into the iOS world.
iOS uses Double for its pixel locations and time intervals, and iOSDriver maps the Int pixel values directly to Double without a Float in between. 

The denormalizedDistance transformation is moved into `width|heightPercentToPoint`. Thus the PointF is now unused:
```
val denormalizedDistance = PointF(start.x * width, start.y * height)
             .distance(PointF(end.x * width, end.y * height))
```